midenc-tx-kernel/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+use std::sync::Arc;
+
+use miden_assembly::{
+    ast::ModuleKind, library::CompiledLibrary, Assembler, Compile, CompileOptions,
+    DefaultSourceManager,
+};
+
+/// Stubs for the Miden rollup tx kernel
+pub struct MidenTxKernelLibrary(CompiledLibrary);
+
+impl AsRef<CompiledLibrary> for MidenTxKernelLibrary {
+    fn as_ref(&self) -> &CompiledLibrary {
+        &self.0
+    }
+}
+
+impl From<MidenTxKernelLibrary> for CompiledLibrary {
+    fn from(lib: MidenTxKernelLibrary) -> Self {
+        lib.0
+    }
+}
+
+impl Default for MidenTxKernelLibrary {
+    fn default() -> Self {
+        // TODO: Load compiled MASL from file
+        let source_manager = Arc::new(DefaultSourceManager::default());
+        let assembler = Assembler::new(source_manager.clone());
+        let tx_module = (include_str!("../masm/tx.masm"))
+            .compile_with_options(
+                source_manager.as_ref(),
+                CompileOptions {
+                    warnings_as_errors: assembler.warnings_as_errors(),
+                    ..CompileOptions::new(ModuleKind::Library, "miden::tx").unwrap()
+                },
+            )
+            .unwrap();
+        let account_module = (include_str!("../masm/account.masm"))
```

Comment:
```suggestion
        let account_source = source_manager.load("midenc-tx-kernel/masm/account.masm", include_str!("../masm/account.masm"));
        let account_module = account_source
```

Reply:
Done in https://github.com/0xPolygonMiden/compiler/pull/277/commits/ab2b7257965b0f6e56d75e84f403d3b9fd683c4b